src/encoded/static/libs/compat.js

```diff
@@ -1,56 +1,39 @@
-/*jshint strict:false */
-require("core-js/stable");
-require("regenerator-runtime/runtime");
+/* jshint strict:false */
+require('core-js/stable');
+require('regenerator-runtime/runtime');
 
 // Chrome 42 fetch does not have abort.
 window.fetch = undefined;
 require('whatwg-fetch');
 
-(function () {
```

Comment:
Immediately Involved function expression (IIFE) no longer needed in ES2015 and higher. Like see - https://stackoverflow.com/a/33534601/178550